### PR TITLE
[gyb] add the moral equivalent of an #include directive 

### PR DESCRIPTION
**Pull request by dabrahams**
The few places that are explicitly using execute_template could benefit

### CODE REVIEW

**Comment by dabrahams** [comment]
@swift-ci Please test

File: utils/gyb.py [hunk 1]
```diff
@@ -554,10 +554,9 @@ class ExecutionContext(object):
 
     """State we pass around during execution of a template"""
 
-    def __init__(self, line_directive='// ###sourceLocation',
-                 **local_bindings):
+    def __init__(self, line_directive=None, **local_bindings):
         self.local_bindings = local_bindings
-        self.line_directive = line_directive
+        self.line_directive = '// ###sourceLocation' if line_directive is None else line_directive
```
Comment (hughbe) [inline utils/gyb.py]: Over 80 chars - will fail python linting

Comment (dabrahams) [inline utils/gyb.py]: Thanks; I'll clean that up before merging it.

**Comment by dabrahams** [comment]
@swift-ci Please smoke test and merge

**Comment by dabrahams** [comment]
@swift-ci Please smoke test and merge

**Pull request merged**